### DISCUSSION TITLE
support for pi1541?

### DISCUSSION THREAD

**Discussion by vejinnovem** [keirf/FF_OSD#35] [Ideas]
Hello. I hope I won't be received as rude.
I'm looking for similiar resolution for pi1541, as it also uses I2C display.
Unfortunately I'm not able to use this code to match the hardware.
Do you think that would be possible for you in future?

**Comment by penfold42** [comment]
It’s not impossible - I’ve considered looking into this myself 

The OSD part should “just work”

the I2C protocols won’t - the pi1541 only does SSD1306 and IIIRC this is more like a HD44780 protocol.

The Amiga keyboard support isn’t very helpful!
How important is remote buttons ? It won’t be easy on the 64 as you need to sniff 16 pins

**Reply by vejinnovem** [reply]
Actually I don't care about keyboard at all.
Only wanted to see currently mapped disk images on screen.
